### PR TITLE
Network Security Groups (NSGs) validation

### PR DESCRIPTION
**Pull request by DBAlberto**
This will help to identify if Network traffic between the Azure VMs through the ports chosen for Database mirroring endpoint, SQL listener, load balancer IP address health probe is allowed

### CODE REVIEW

**Comment by PRMerger15** [comment]
@DBAlberto : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: azure-sql/virtual-machines/windows/availability-group-manually-configure-tutorial-single-subnet.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ The following table lists the prerequisites that you need to complete before sta
...
 |:::image type="icon" source="./media/availability-group-manually-configure-tutorial-single-subnet/square.png" border="false":::   **SQL Server Agent service account** | Domain account |  
 |:::image type="icon" source="./media/availability-group-manually-configure-tutorial-single-subnet/square.png" border="false":::   **Firewall ports open** | - SQL Server: **1433** for default instance <br/> - Database mirroring endpoint: **5022** or any available port <br/> - Availability group load balancer IP address health probe: **59999** or any available port <br/> - Cluster core load balancer IP address health probe: **58888** or any available port |
+|:::image type="icon" source="./media/availability-group-manually-configure-tutorial-single-subnet/square.png" border="false":::   **Network Security Groups (NSGs)** | If the environment is using [Network security groups](azure-docs/articles/virtual-network/network-security-groups-overview.md), ensure that the current configuration **allows Network traffic** through ports described in the “Firewall ports open” section. |
```
Comment (Jak-MS) [inline azure-sql/virtual-machines/windows/availability-group-manually-configure-tutorial-single-subnet.md:53]:
```suggestion
|:::image type="icon" source="./media/availability-group-manually-configure-tutorial-single-subnet/square.png" border="false":::   **Network Security Groups (NSGs)** | If the environment is using [Network security groups](/azure/virtual-network/network-security-groups-overview), ensure that the current configuration **allows Network traffic** through ports described in the “Firewall ports open” section. |
```
fixing link

**Comment by Jak-MS** [comment]
@adbadram 

Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by tarynpratt** [comment]
#reassign:tarynpratt

**Comment by tarynpratt** [comment]
#sign-off

**Comment by MashaMSFT** [comment]
Hi @DBAlberto , thanks so much for your submission and our apologies for the delay! Due to the merge conflict, I've pulled your change into the private repo, and am going to close out this PR. Your change will still go live and you'll still be marked as a contributor to the doc :) This change should be live by next week. 

Thanks again for improving our docs and I hope you have a wonderful rest of your day!
Masha from the SQL Docs team

**Pull request closed without merging**